docs/tools/Ide/vs-code.md

```diff
@@ -1,3 +1,66 @@
-# VS Code
+# KCL Extension for Visual Studio Code
 
-+ VSCode Plugin: https://github.com/KusionStack/vscode-kcl
+The [VS Code KCL extension](https://marketplace.visualstudio.com/items?itemName=kcl.kcl-vscode-extension) provides rich language support for the [KCL language](https://kcl-lang.io/).
+
+## Quick Start
+
+Welcome to KCL developing! We hope this extension enhances your development experience with KCL.
+
+- **Step 1.** Install KCL on your system.
+  - If you are a Kusion user, it is recommended to install the [VS Code Kusion extension](https://marketplace.visualstudio.com/items?itemName=KusionStack.kusion). It will provide more rich features and guide you to install Kusion.
+  - If you just want to use KCL alone, you need to [install KCL](https://kcl-lang.io/docs/user_docs/getting-started/install)
+
+  Please check that `kcl` and `kcl-language-server` are installed and have been added to Path
+
+  ```bash
+  which kcl
+  which kcl-language-server
+  ```
+
+- **Step 2.** Install the KCL extension for Visual Studio Code. This extension requires the VS Code 1.50+
```

Comment:
Add the install link. [KCL extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=kcl.kcl-vscode-extension)